common/src/main/java/com/wynntils/models/profession/ProfessionModel.java

```diff
@@ -22,41 +21,33 @@
 import com.wynntils.models.profession.type.ProfessionType;
 import com.wynntils.models.worlds.BombModel;
 import com.wynntils.models.worlds.WorldStateModel;
-import com.wynntils.models.worlds.event.WorldStateEvent;
-import com.wynntils.models.worlds.type.BombType;
 import com.wynntils.utils.mc.LoreUtils;
-import com.wynntils.utils.mc.McUtils;
-import com.wynntils.utils.mc.RenderedStringUtils;
+import com.wynntils.utils.type.Pair;
 import com.wynntils.utils.type.TimedSet;
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import net.minecraft.world.entity.Entity;
-import net.minecraft.world.entity.EntityType;
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class ProfessionModel extends Model {
-    // §7[+36§f Ⓙ§7 Farming XP] §6[9%]
-    // §dx2.0 §7[+§d93§f Ⓙ§7 Farming XP] §6[9%]
+    // §7x1 [+3952§f Ⓒ§7 Woodcutting XP] §6[14.64%]
     private static final Pattern PROFESSION_NODE_EXPERIENCE_PATTERN = Pattern.compile(
-            "(§dx[\\d\\.]+ )?§7\\[\\+(§d)?(?<gain>\\d+)§f [ⓀⒸⒷⒿⒺⒹⓁⒶⒼⒻⒾⒽ]§7 (?<name>.+) XP\\] §6\\[(?<current>\\d+)%\\]");
+            "(§.x[\\d\\.]+ )?(§.)?\\[\\+(§d)?(?<gain>\\d+)§f [ⓀⒸⒷⒿⒺⒹⓁⒶⒼⒻⒾⒽ]§7 (?<name>.+) XP\\] §6\\[(?<current>[\\d.]+)%\\]");
```

Comment:
Can't verify this works with x2 XP but if it doesn't can always do a follow up fix

Reply:
Yes, i would assume it does though

Reply:
As luck would have it I just happened to get in a world with one and can confirm it works